charts/argoconfig/README.md

```diff
@@ -16,18 +16,19 @@ Source code can be found [here](https://github.com/adfinis-sygroup/helm-charts)
 | appProject.clusterResourceWhitelist[0].kind | string | `"*"` | API kind to whitelist |
 | appProject.destinations[0].namespace | string | `"*"` | Destination namespace to allow |
 | appProject.destinations[0].server | string | `"*"` | Destination server to allow |
-| appProject.enabled | bool | `true` | Enable creating an AppProject resource |
+| appProject.enabled | bool | `false` | Enable creating an AppProject resource |
 | appProject.namespace | string | `"argocd"` | Namespace for AppProject resource |
 | appProject.sourceRepos | list | `["*"]` | Source repos to allow |
-| application.destination.namespace | string | `"default"` | Target namespace of application in Argo CD |
-| application.destination.server | string | `"https://kubernetes.default.svc"` | Target cluster in Argo CD |
-| application.enabled | bool | `true` | Enable creating an Application resource |
-| application.namespace | string | `"argocd"` | Namespace for Application resource |
-| application.projectOverride | string | Fullname | Name of AppProject to install into |
-| application.source.chart | string | `""` | Name of chart in source repo |
-| application.source.repoURL | string | `""` | URL of sourece repo |
-| application.source.targetRevision | string | `"*"` | Revision of chart in source repo to install |
-| application.syncPolicy | object | deactivated | Application [Sync Policy](https://argoproj.github.io/argo-cd/user-guide/auto_sync/) |
+| apps[0].destination.namespace | string | `"default"` | Target namespace of application in Argo CD |
+| apps[0].destination.server | string | `"https://kubernetes.default.svc"` | Target cluster in Argo CD |
+| apps[0].enabled | bool | `true` | Enable creating an Application resource |
+| apps[0].name | string | `"app"` | Name of app, used as suffix to app specific resources |
+| apps[0].namespace | string | `"argocd"` | Namespace for Application resource |
+| apps[0].projectOverride | string | Fullname | Name of AppProject to install into |
+| apps[0].source.chart | string | `""` | Name of chart in source repo |
+| apps[0].source.repoURL | string | `""` | URL of sourece repo |
```

Comment:
shouldn't we also allow having an `apps[0].source.path` instead of repoURL? There might be the need for adding apps which aren't released on any charts repository, but are only available through git.

Reply:
This is just and example, it uses `{{- .source | toYaml | nindent 4 }}` under the hood to allow expressing anything that is a valid Argo CD Application.

Reply:
Yes I see, but it's also the README which holds all possible values, right? shouldn't we document it?

Reply:
It's mostly in the README so there is an example in there. In this case the README isn't trying to be complete since it doesn't want to contain everything that argo allows.

Reply:
I ended up refactoring this to mostly link upstream docs and only mention things that are specific to this chart. The README is now small and sweet :)